AElf.Kernel.Tests/Concurrency/Scheduling/ParallelTestDataUtil.cs

```diff
@@ -309,57 +307,5 @@ public HashSet<Resource> TranslateStringToResourceSet(string function, string[][
             }).ToHashSet();
 
         }
-        
-        public string FunctionMetadataMapToString(Dictionary<string, FunctionMetadata> map)
```

Comment:
Also remove the `XXXToString` methods. They are used in unit tests to compare two HashSet equals. However, if we correctly implement `Equal` method, they can be removed.